h5p-server/renderers/player.js

```diff
@@ -0,0 +1,69 @@
+export function playerRenderer(model) {
+  var context = "";
+
+  function playerInit(contentId = "") {
+    H5P.preventInit = true;
+
+    if (H5PIntegration) {
+      let materiaPath = "";
+      switch (context) {
+        case "prod":
+        case "dev":
+          materiaPath = "https://localhost:8008";
```

Comment:
This url definitely shouldn't be hard-coded, but rather sourced from an env or other configuration. Doing so usually means pushing an inline variable definition in the fuel controller method that renders this content, via:

```
Js::push_inline('var BASE_URL = "'.Uri::base().'";');
```
Here's an example: https://github.com/ucfopen/Materia/blob/master/fuel/app/classes/controller/widgets.php#L519